R/densityClust.R

```diff
@@ -69,6 +69,8 @@ NULL
 localDensity <- function(distance, dc, gaussian=FALSE) {
    # These implementations are faster by virtue of being written in C++
    # They also avoid the need to convert `distance` to a matrix. 
+   message(paste("the length of the distance:", length(distance)))
```

Comment:
You don't need to wrap it in paste - message does that for you

This goes for all the verbose messaging